source/server-discovery-and-monitoring/server-monitoring.rst

```diff
@@ -877,6 +877,23 @@ polling protocol as well as consistency with the initial RTT provided the
 connection handshake which also uses the isMaster command. Additionally,
 mongocryptd does not allow the ping command but does allow isMaster.
 
+Why not use `awaitedTimeMS` in the server response to calculate RTT in the streaming protocol?
+''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+One approach to calculating RTT in the streaming protocol would be to have the server
+return an `awaitedTimeMS` in its `isMaster` response. A driver could then determine the
+RTT by calculating the difference between the initial request, or last response, and the
+`awaitedTimeMS`.
+
+We rejected this design because of issues with clock skew between local and remote
+system clocks. This approach mixes two notions of time: the local clock times the whole
+operation while the remote clock times the wait. This means that if these clocks tick
+at different rates, or there are anomalies like clock changes, you will get bad results.
+To make matters worse, you will be comparing times from multiple servers that could each
+have clocks ticking at different rates. This approach will bias toward servers with the
+slowest ticking clock, since it will seem like it spends the least time on the wire.
```

Comment:
I thought it would bias towards the _fastest_ ticking clock? If the server clock is ticking faster, then it may say it waited 10000ms even though it only waited 9500ms in "true time". That would make the server have a 500ms _lower_ RTT.

Reply:
Agreed, I switched it to `fastest`